conda_forge_tick/auto_tick.py

```diff
@@ -0,0 +1,930 @@
+import copy
+import glob
+import json
+import time
+import traceback
+import logging
+import typing
+
+from urllib.error import URLError
+
+import github3
+import ruamel.yaml as yaml
+from uuid import uuid4
+
+from conda_forge_tick.contexts import FeedstockContext
+from .git_utils import (
+    get_repo,
+    push_repo,
+    is_github_api_limit_reached,
+    ensure_label_exists,
+    label_pr,
+)
+from .path_lengths import cyclic_topological_sort
+from .utils import (
+    setup_logger,
+    pluck,
+    get_requirements,
+    load_graph,
+    dump_graph,
+    LazyJson,
+)
+from .xonsh_utils import env, eval_xonsh
+from typing import MutableSequence, Sequence, Tuple, Dict, Set, Sized
+
+logger = logging.getLogger("conda_forge_tick.auto_tick")
+
+# TODO: move this back to the bot file as soon as the source issue is sorted
+# https://travis-ci.org/regro/00-find-feedstocks/jobs/388387895#L1870
+from .migrators import *
+
+
+MIGRATORS: MutableSequence[Migrator] = [
+    Version(pr_limit=30, piggy_back_migrations=[PipMigrator(), LicenseMigrator()]),
+    # Noarch(pr_limit=10),
```

Comment:
I would pull these out, since they haven't been used in a long time and are unlikely to be reactivated.